.changeset/yellow-goats-travel.md

```diff
@@ -2,4 +2,95 @@
 '@openfn/language-fhir': major
 ---
 
-`fhir` redesign
+### Added
+
+- Introduced a new `RequestOptions` for the HTTP request:
+
+  - `headers`: An object of headers to append to the request.
+  - `body`: JSON payload to attach to the request.
+  - `query`: Query parameters for the request, encoded into the URL.
+  - `errors`: Map of errorCodes -> error messages, ie,
+    `{ 404: 'Resource not found;' }`. Pass `false` to suppress errors for this
+    code.
+  - `timeout`: Request timeout in ms. Default: 300 seconds.
+
+- Introduced a new `GetOptions` for the GET HTTP request:
+
+  - `headers`: An object of headers to append to the request.
+  - `errors`: Map of errorCodes -> error messages, ie,
+    `{ 404: 'Resource not found;' }`. Pass `false` to suppress errors for this
+    code.
+  - `timeout`: Request timeout in ms. Default: 300 seconds.
+
+- Introduced a new `FHIRHttpState` for the State object:
```

Comment:
Again, the state annotations are pure docs. At the most we can say "Added state mutation tables", but I don't even think we should add that.